keyboards/contra/keymaps/deastiny/config.h

```diff
@@ -0,0 +1,42 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the boiler plate: 
```c
#ifndef CONFIG_USER_H
#define CONFIG_USER_H

#include "config_common.h"
```
with just `#pragma once`, and remove the endif at the bottom?

Reply:
so just delete all of it and only write the pragma part?

Reply:
Yes, delete those three lines (and the one at the end), and replace it with `#pragma once`.